src/app/components/Article.jsx

```diff
@@ -38,6 +46,8 @@ class Article extends Component {
         <Helmet htmlAttributes={{ lang: 'en-GB' }}>
           <title>{headline}</title>
         </Helmet>
+        <Body />
```

Comment:
This `Body` element isn't doing anything. Is it meant to?

Reply:
You're adding a second `<body>` element to the page with this. If you want to reset the styles, it'd be best to use a reset or normalise CSS file.
https://www.npmjs.com/package/styled-reset
https://www.npmjs.com/package/styled-normalize
This is outside the scope of this ticket, though - we'll need to discuss as a team whether we'll want to do things this way.

Reply:
@sareh @jtart Chat after stand-up?